src/util.jl

```diff
@@ -33,3 +35,22 @@ function minimal_activity(ps::PresolveData{T}, i::Int)::T where {T}
     end
     return inf
 end
+
+# In some presolve procedures, when we process rows, we need to update
+# columns accordingly. This function performs synchronization between
+# rows and columns after a presolve method is applied on every rows.
+function sync_columns_to_rows!(ps::PresolveData{T}) where {T}
```

Comment:
Unit test?

Reply:
In my cg_strenthening_test1, 2, 3, I used this function to sync column after calling to cg_strengthening!. Is it not enough for unit test ?

Reply:
Not for a unit test :)

A unit test should be a simple test of _only_ the target functionality, isolated from other bits of code, where you have complete control over the inputs. Right now, since you only call `sync_columns_to_rows` right after a call to `cg_strengthening!`, a breakage in the second could mask a breakage in the first.

Also, since this functionality lives in different piece of the codebase, it should be tested in a corresponding location in the `test/` directory, not tucked away in an otherwise unrelated file.